### PR TITLE
24 admin viewing items

### PR DESCRIPTION
**Pull request by JoelLindow**
Related Issue: #4 

Summary: Table for /admin/items with link to edit item

Author(s): Joel and Randy-Bandy-Bo-Bandy-Banana-Fana-Fo-Fandy_Mi-Mi-Mo-Mandy....Randy

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -90,3 +90,7 @@ body > div > div.metrics-wrapper > div {
 .dashboard-wrapper > h1 {
   margin-left: 5px;
 }
+.admin-item-thumb {
```
Comment (randy-darsh) [inline app/assets/stylesheets/application.css:93]: i like it

File: app/controllers/admin/items_controller.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class Admin::ItemsController < Admin::BaseController
+  def index
+    @items = Item.all
```
Comment (randy-darsh) [inline app/controllers/admin/items_controller.rb:3]: this was essential

File: spec/features/admin/admin_can_see_all_items_and_click_edit_on_admin_show_view_spec.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+# -And when I click that link
+# -Then my current path should be “/admin/items”
+# -Then I should see a table with all items (active and inactive)
+# -And each item should have:
+#
+# A thumbnail of the image
+# Title that links to the item
+# Description
+# Status
+# Actions (Edit)
```
Comment (randy-darsh) [inline spec/features/admin/admin_can_see_all_items_and_click_edit_on_admin_show_view_spec.rb]: remember to remove these later

File: app/views/admin/items/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      <th><img class="admin-item-thumb" src="<%= item.image %>" alt=""></th>
+      <td><%= item.title %></td>
+      <td><%= item.description %></td>
+      <td><%= item.price %></td>
+      <td><%= item.status %></td>
+      <td><%= link_to "Edit", edit_admin_item_path(item) %></td>
+    </tr>
+    <% end %>
+  </tbody>
+</table>
```
Comment (randy-darsh) [inline app/views/admin/items/index.html.erb:25]: table looks great

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -90,3 +90,7 @@ body > div > div.metrics-wrapper > div {
...
+  width: 100px;
```
Comment (limsammy) [inline app/assets/stylesheets/application.css:94]: Good size.

**Review by limsammy — APPROVED** [review]
Looks good, merging

**Pull request merged**